### ISSUE TITLE
Dropcap missing on some articles

### ISSUE THREAD

**Issue by alexgreen** [TheTyee/design-article.thetyee.ca#274]
e.g. 
http://preview.thetyee.ca/Presents/2016/06/09/Start-Farmers-Market/index.html
http://preview.thetyee.ca/News/2015/12/17/Peace-Valley-Construction/index.html

**Comment by phillipadsmith** [comment]
> e.g. http://preview.thetyee.ca/Presents/2016/06/09/Start-Farmers-Market/index.html

I believe that's because the paragraph is too short (in characters) to support the drop cap visually, i.e., it will overhang the paragraph.

**Comment by alexgreen** [comment]
That paragraph occupies 3 lines, so it's long enough to support a drop cap.

I'm also seeing this issue on http://preview.thetyee.ca/Culture/2016/07/07/Try-Bibliotherapy/index.html 

And as described in #202 : the styling should be accomplished by extending the existing .lead-in class rather than by eliminating it. That way it would be possible to target the object via css.

**Comment by phillipadsmith** [comment]
> That paragraph occupies 3 lines, so it's long enough to support a drop cap.

I believe the cut off is character-based, not based on lines (which would vary according to screen width). I believe the cut off is 240 or 250 characters, which both of these are under.

> And as described in #202 : the styling should be accomplished by extending the existing .lead-in class rather than by eliminating it. That way it would be possible to target the object via css.

Makes sense. Let's circle back to this post-launch.

**Comment by MrBryan** [comment]
As I dove into this it's quite complicated.

I have been experimenting with this today -- We can get at the number of lines in a somewhat hacky way by diving the height of an element by line-height of the text ( the fact we are using ems complicates that slightly)

But if the drop-cap is there, it will not let us measure the height of the wrapping element because it will have bumped it up to the minimum 75px already.  So we can add the class only after pageload but then if it is resized we'd have to first remove the class then measure height (sans dropcap) then add it back and there isn't a good way of hiding that weirdness from the user where the dropcap would shrink and enlarge / disappear / reappear.

Trying from another angle -- the width of the container should predict how many lines it will spill over to but then we have some unpredictability from word length and lines breaks and also that elements such as the inline image might change where text flows.

I started on a way to wrap the first and last letters in the paragraph in spans, then measure the difference of their offsets and add the line-height and that works for that part thus far although it is a bit much and the problem of adding and removing before testing remains.

In short it's a bit of a bear to get this working.

I think we should just leave as-is. Only a small minority of pages have it omitted and editors can preview and decide if they want to lengthen the paragraphs.

I can still use the paragraph length to determine which classes it will have but when i use your class extension at the moment it still looks like this: 

![image](https://cloud.githubusercontent.com/assets/1721891/17005818/5d643d98-4eaa-11e6-8a18-400fd8be3c6c.png)
